docs/download/77.md

```diff
@@ -16,15 +16,14 @@ License & terms of use: http://www.unicode.org/copyright.html
 ICU is the [premier library for software internationalization](https://icu.unicode.org/#h.i33fakvpjb7o),
 used by a [wide array of companies and organizations](https://icu.unicode.org/#h.f9qwubthqabj).
 
-## Release Candidate
-
-**Please use this release candidate for testing, but do not use it in production!**
-
 ## Release Overview
 
+* Download: [releases/tag/release-77-1](https://github.com/unicode-org/icu/releases/tag/release-77-1)
+* [Maven: com.ibm.icu / icu4j / version 77.1](https://mvnrepository.com/artifact/com.ibm.icu/icu4j/77.1)
+
 ICU 77 updates to
 [CLDR 47](https://cldr.unicode.org/downloads/cldr-47)
-([alpha blog](https://blog.unicode.org/2025/02/unicode-cldr-47-alpha-now-available-for.html))
+([beta blog](https://blog.unicode.org/2025/02/unicode-cldr-47-beta-available-for.html))
```

Comment:
The only nitpick: do we want to point to the beta blog? When we know the final will be out soon(ish)?

M

Reply:
I can't point to the release blog post until there is one...